sds/src/secondary_validation.rs

```diff
@@ -92,6 +96,32 @@ impl Validator for ChineseIdChecksum {
     }
 }
 
+impl Validator for GithubTokenChecksum {
+    fn is_valid_match(&self, regex_match: &str) -> bool {
+        // Implementation of https://github.blog/2021-04-05-behind-githubs-new-authentication-token-formats/
+        let parts: Vec<&str> = regex_match.split('_').collect();
+        if parts.len() < 2 {
+            return false;
+        }
+        let last_part = parts[parts.len() - 1];
```

Comment:
parts.last() is more idiomatic here (It returns Option<&str>)
```suggestion
        let last_part = parts.last().unwrap();
```